_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,42 @@
+
+
+# Something New:
+
+There is only 8 months, 7 days, and 6 hours until check-in, but who’s counting? HackIllinois 2019 will be the 6th time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. We have established our mission this year:
+
+**_To create an environment conducive to education, collaboration, and mentorship within the Open Source Community_**
+
+We _create an enviroment_  in that our event is not defined by a simple "code for a prize" model. We plan to pack the entire event with content for attendees, from UIUC and over a dozen other universities, to have a unique experience throughout the weekend. HackIllinois carries development, social, and networking responsibilities, and these establish the three pillars —  _education, collaboration, and mentorship_ — of our event. The theme this year follows the trail that has been set forth by our past directors and staff, but we wanted to assert our post _within the Open Source community_. We fully believe that our event betters the Open Source Community: by bringing in new students, continuing development on projects we believe in, and showcasing the talent pool that we have to offer.
+
+More specifically, this year we want to make HackIllinois **exciting**. We are putting as much emphasis as we can on making this event more than 36 hours of coding; learning about open source is not enough, in 2019. It will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We will run interactive workshops, present lively mini-events, and host top industry talent for our weekend in February. The Open Source experience will be unique from the past: exciting to both the OSS gurus and those who have not yet explored this region of the technology world.
+
+
+## Something Borrowed:
+
+We are always open to feedback and new ideas. If you have participated in our event or any hackathon, conference, get-together, Urbana corn festival, ANYTHING where you found a fun or engaging project, activity, or meal, do not hesitate to suggest it to us. We are still in the brainstorming phases for the event and love to hear from you all, so reach out to us! Working together and getting advice from informed individuals is our style. #(OUR HASHTAG HERE)
+
+There are some pilot ideas we will be exploring over the following months as planning continues, so let us know on contact@hackillinois.org if you have suggestions, feedback, or just want to share your experiences with us. We’re always happy to hear from you! We will continue to make more announcements on our blog, right here, and with our social media accounts.
```

Comment:
Like Spencer said, you ask for feedback a ton haha

"make more announcements on our blog and social media accounts"
right here is kind of an awkward clause in there?